### PR TITLE
Technology component

### PR DESCRIPTION
**Pull request by fededirocco**
## Sumary
* Implemented technology component
* Added div for this component in index.html
* Added scss for this component

## Trello Card
https://trello.com/c/FAQQsrJX/22-tecnolog%C3%ADa

## Screenshots
technology-component

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,7 +10,17 @@
 </head>
 <body>
   <div class="title">
-      <h1 class="title-name">Front-End Tools</h1>
-  </div>  
+    <h1 class="title-name">Front-End Tools</h1>
+  </div>
+  <div class="tech">
```
Comment (jjuli93) [inline index.html]: Is this div neccesary? can't we group it with its only child?

File: scss/tech.scss [hunk 2]
```diff
@@ -0,0 +1,43 @@
+.tech {
+  display: flex;
+  flex-wrap: wrap;
```
Comment (jjuli93) [inline scss/tech.scss]: why flex wrap?

File: scss/tech.scss [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  justify-content: space-around;
```
Comment (jjuli93) [inline scss/tech.scss]: I think this isn't doing anything

File: scss/tech.scss [hunk 4]
```diff
@@ -0,0 +1,43 @@
+.tech {
```
Comment (jjuli93) [inline scss/tech.scss]: I think this class is not neccesary yet

File: scss/tech.scss [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  padding: 50px;
+  width: 100%;
+
+  .box {    
+    align-items: center;
+    background-color: $green;
+    border: 2px;
+    border-radius: 5px;
+    box-shadow: -10px -10px 0 0 white, -10px -10px 0 3px $green;
+    display: inline-flex;
```
Comment (jjuli93) [inline scss/tech.scss]: why not only flex?

File: index.html [hunk 6]
```diff
@@ -10,7 +10,17 @@
...
+    <div class="box">
+      <img class="img" src="img/react_icon.svg" />
+      <div class="text">
+        <p class="name">ReactJS</p>
```
Comment (jjuli93) [inline index.html]: This should be a tittle

File: scss/tech.scss (hunk 2 above)
Comment (fededirocco) [reply scss/tech.scss]: I dont know why I put but I removed it and everything goes ok

File: scss/tech.scss (hunk 5 above)
Comment (fededirocco) [reply scss/tech.scss]: Changed to flex and it remains the same (don't remember why I used inline-flex)

File: index.html (hunk 1 above)
Comment (fededirocco) [reply index.html]: Yes, its not necessary in this task. I done it thinking in next implementations.

File: index.html [hunk 7]
```diff
@@ -10,7 +10,15 @@
...
   <div class="title">
-      <h1 class="title-name">Front-End Tools</h1>
-  </div>  
+    <h1 class="title-name">Front-End Tools</h1>
+  </div>
+  <div class="box">
+    <img class="img" src="img/react_icon.svg" />
+    <div class="text">
+      <h1 class="name">ReactJS</h1>
+      <p class="release"><strong>Release: 2013</strong></p>
```
Comment (FrankIglesias) [inline index.html]: Don't use html tags for styling

File: scss/tech.scss [hunk 8]
```diff
@@ -0,0 +1,35 @@
+.box {    
+  align-items: center;
+  background-color: $green;
+  border: 2px;
+  border-radius: 5px;
+  box-shadow: -10px -10px 0 0 white, -10px -10px 0 3px $green;
```
Comment (FrankIglesias) [inline scss/tech.scss]: use a variable color for this

File: scss/tech.scss [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+  display: flex;
+  flex-direction: column;
+  padding: $padding-medium;
+  width: 200px;
+
+  .img {
```
Comment (FrankIglesias) [inline scss/tech.scss]: use a more descriptive name

File: scss/tech.scss [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+
+    .release {
+      margin: $padding-small;
+    }
+
+    .desc {
+      margin: $padding-big;
+    }
+  }
+}
```
Comment (FrankIglesias) [inline scss/tech.scss]: add empty line

File: scss/tech.scss [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+
+  .text {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    font-size: $text-medium;
+    text-align: center;
+  
+    .name {
+      margin: $padding-small;
```
Comment (FrankIglesias) [inline scss/tech.scss]: Why you name this variable as padding if you are using it for margin

File: scss/tech.scss [hunk 12]
```diff
@@ -0,0 +1,35 @@
...
+    }
+
+    .release {
+      margin: $padding-small;
```
Comment (FrankIglesias) [inline scss/tech.scss]: same

File: scss/tech.scss [hunk 13]
```diff
@@ -0,0 +1,35 @@
...
+    }
+
+    .desc {
+      margin: $padding-big;
```
Comment (FrankIglesias) [inline scss/tech.scss]: same

File: scss/tech.scss [hunk 14]
```diff
@@ -0,0 +1,35 @@
+.box {    
+  align-items: center;
+  background-color: $green;
+  border: 2px;
```
Comment (FrankIglesias) [inline scss/tech.scss]: Is this border required?

File: index.html [hunk 15]
```diff
@@ -10,7 +10,15 @@
 </head>
 <body>
   <div class="title">
-      <h1 class="title-name">Front-End Tools</h1>
-  </div>  
+    <h1 class="title-name">Front-End Tools</h1>
+  </div>
+  <div class="box">
+    <img class="img" src="img/react_icon.svg" />
```
Comment (FrankIglesias) [inline index.html]: Add an alt here

**Pull request merged**